### ISSUE TITLE
2.0.5 SOAP API: Encoding: object has no 'label' property

### ISSUE THREAD

**Issue by mch35** [magento/magento2#4630]
Hello,

I try to use Magento 2.0.5 SOAP API. I am not able to retrieve attribute options. Attribute with code dm1 exists in the shop.

Request

``` XML
<soap:Envelope xmlns:soap="http://www.w3.org/2003/05/soap-envelope" xmlns:def="XXX/soap/default?services=catalogProductAttributeOptionManagementV1%2CintegrationAdminTokenServiceV1">
   <soap:Header/>
   <soap:Body>
      <def:catalogProductAttributeOptionManagementV1GetItemsRequest>
         <attributeCode>dm1</attributeCode>
      </def:catalogProductAttributeOptionManagementV1GetItemsRequest>
   </soap:Body>
</soap:Envelope>
```

Response

``` XML
<env:Envelope xmlns:env="http://www.w3.org/2003/05/soap-envelope">
   <env:Body>
      <env:Fault>
         <env:Code>
            <env:Value>env:Receiver</env:Value>
         </env:Code>
         <env:Reason>
            <env:Text>SOAP-ERROR: Encoding: object has no 'label' property</env:Text>
         </env:Reason>
      </env:Fault>
   </env:Body>
</env:Envelope>
```

**Comment by SteveRohrlack** [comment]
same here

As a reference, I created a dump from [catalogProductAttributeRepositoryV1Get](https://github.com/magento/magento2/files/277706/catalogProductAttributeRepositoryV1Get.txt) for the attribute.

At first I thought this has something to do with the option labels per store but all options already have separate labels for store and admin.

**Comment by SteveRohrlack** [comment]
same here

As a reference, I created a dump from [catalogProductAttributeRepositoryV1Get](https://github.com/magento/magento2/files/277706/catalogProductAttributeRepositoryV1Get.txt) for the attribute.

At first I thought this has something to do with the option labels per store but all options already have separate labels for store and admin.

**Comment by choukalos** [comment]
Tracking internally - MAGETWO-53431

**Comment by magento-engcom-team** [comment]
@mch35, thank you for your report.
The issue is already fixed in 2.2.0
